tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -305,6 +311,15 @@ func testInstanceIsAssociatedWithClusterWorkerIAM(t *testing.T) {
 	assert.Equal(t, *iamProfile.Arn, *createdInstance.IamInstanceProfile.Arn, "instance is not associated with worker IAM profile")
 }
 
+// testInstanceFirewallRule asserts if the created instance has firewall rule that enable console logs
+func testAWSInstanceFirewallRule(t *testing.T) {
+	winRMClient, err := setupWinRMClient(credentials.GetIPAddress(), credentials.GetPassword(), awsUser)
```

Comment:
I had requested that this be moved to a common setup section as there is potential for this be used in multiple tests.